css/index.css

```diff
@@ -0,0 +1,62 @@
+
+body {
```

Comment:
You probably want to create this file inside the instructions folder just like you did with the index.html file and the js folder, otherwise the imports get screwy.